electron-app/src/components/settings/PremiumSettings.vue

```diff
@@ -67,20 +78,31 @@
         </v-card-actions>
       </v-card>
     </v-col>
+    <confirm-dialog
+      :display="confirmDeletePremium"
+      confirm-type="warning"
+      primary-action="Delete"
+      title="Delete rotki premium keys?"
+      message="Are you sure you want to delete your rotki premium keys? Premium will be disabled for this session and if you want to re-enable premium you will have to enter your keys again."
```

Comment:
> will be disabled for this session

Maybe change it to something more permanent? "for your account"? The current wording made me think it's temporary only for the session.

Reply:
Done.